From 81ef355c419376cfce86566e19431178c53647bd Mon Sep 17 00:00:00 2001
From: Pavel Asyutchenko <svenpavel@gmail.com>
Date: Sun, 2 Mar 2025 00:28:24 +0100
Subject: [PATCH] Patch CMakeLists.txt for Voxen

---
 CMakeLists.txt | 18 ------------------
 1 file changed, 18 deletions(-)

diff --git a/CMakeLists.txt b/CMakeLists.txt
index 00c05908..7ebe62bc 100644
--- a/CMakeLists.txt
+++ b/CMakeLists.txt
@@ -272,24 +272,6 @@ if (MSVC)
   set(WERROR_FLAG /WX)
 endif ()
 
-if (FMT_MASTER_PROJECT AND CMAKE_GENERATOR MATCHES "Visual Studio")
-  # If Microsoft SDK is installed create script run-msbuild.bat that
-  # calls SetEnv.cmd to set up build environment and runs msbuild.
-  # It is useful when building Visual Studio projects with the SDK
-  # toolchain rather than Visual Studio.
-  include(FindSetEnv)
-  if (WINSDK_SETENV)
-    set(MSBUILD_SETUP "call \"${WINSDK_SETENV}\"")
-  endif ()
-  # Set FrameworkPathOverride to get rid of MSB3644 warnings.
-  join(netfxpath
-       "C:\\Program Files\\Reference Assemblies\\Microsoft\\Framework\\"
-       ".NETFramework\\v4.0")
-  file(WRITE run-msbuild.bat "
-    ${MSBUILD_SETUP}
-    ${CMAKE_MAKE_PROGRAM} -p:FrameworkPathOverride=\"${netfxpath}\" %*")
-endif ()
-
 function(add_headers VAR)
   set(headers ${${VAR}})
   foreach (header ${ARGN})
-- 
2.48.1

